src/pudl/helpers.py

```diff
@@ -1085,12 +1099,16 @@ def zero_pad_zips(zip_series, n_digits):
 
     """
     # Add preceeding zeros where necessary and get rid of decimal zeros
+    def get_rid_of_decimal(series):
+        return series.str.replace(r'[\.]+\d*', '', regex=True)
+
     zip_series = (
-        pd.to_numeric(zip_series)  # Make sure it's all numerical values
-        .astype(pd.Int64Dtype())  # Make it a (nullable) Integer
-        .fillna(0)  # fill the NA
-        .astype(str).str.zfill(n_digits)  # Make it a string and zero pad it.
-        .astype(pd.StringDtype())  # Make it nullable
+        zip_series
+        .astype(pd.StringDtype())
```

Comment:
did converting these to nullable ints not work here? is this because of the canadian zips?

Reply:
yep!